### PR TITLE
announce fiaf and cna travel grants

### PR DESCRIPTION
**Pull request by dericed**

### CODE REVIEW

**Comment by dericed** [comment]
ping @ablwr @JeromeMartinez for review

File: src/NoTimeToWaitBundle/Resources/views/Default/nttw3.html.twig [hunk 1]
```diff
@@ -53,7 +53,11 @@ Closing: 17:00</p>
 
 <h2>Travel Grants</h2>
 
-<p>(To Be Announced)</p>
+<p>We are pleased to announce four travel grants thanks to </p>
```
Comment (ablwr) [inline src/NoTimeToWaitBundle/Resources/views/Default/nttw3.html.twig]: ???

**Review by ablwr — CHANGES_REQUESTED** [review]

File: src/NoTimeToWaitBundle/Resources/views/Default/nttw3.html.twig (hunk 1 above)
Comment (dericed) [reply src/NoTimeToWaitBundle/Resources/views/Default/nttw3.html.twig]: thx fixed

**Review by ablwr — APPROVED** [review]

**Comment by JeromeMartinez** [comment]
Online.

**Pull request merged**